### PR TITLE
auto login user after registration

### PR DESCRIPTION
**Pull request by miltontony**

### CODE REVIEW

**Comment by miltontony** [comment]
@saeedmb8 @KaitCrawford review please
@saeedmb8 The changes here are likely to conflict with your PR, but you should be able to see what I did and how it fits with yours.

File: molo/profiles/views.py [hunk 1]
```diff
@@ -9,24 +8,27 @@
...
-            user = User.objects.create_user(
-                username=form.cleaned_data['username'],
-                password=form.cleaned_data['password'],
-            )
-            user.profile.date_of_birth = form.cleaned_data['date_of_birth']
-            user.profile.save()
-            return HttpResponseRedirect(form.cleaned_data.get('next', '/'))
-        return render(request, 'profiles/register.html', {'form': form})
-    form = RegistrationForm()
-    return render(request, 'profiles/register.html', {'form': form})
```
Comment (KaitCrawford) [inline molo/profiles/views.py:29]: For my own learning: does the new form_valid method below automatically handle the above cases where the form is invalid or the user has yet to submit it? Is this part of Django's magic?

**Comment by KaitCrawford** [comment]
asked a question for my own learning but otherwise :+1:

File: molo/profiles/views.py (hunk 1 above)
Comment (miltontony) [inline molo/profiles/views.py:29]: Yes it does. All that crap is handled by the Class based view. You can [learn more](https://docs.djangoproject.com/en/1.8/topics/class-based-views/generic-editing/) about it here.

**Comment by miltontony** [comment]
@saeedmb8 please have a look at this PR because it touches on your PR

**Pull request merged**